src/Integration/Laravel/Queue/src/Connector/AsyncAwsSqsConnector.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace AsyncAws\Illuminate\Queue\Connector;
+
+use AsyncAws\Illuminate\Queue\AsyncAwsSqsQueue;
+use AsyncAws\Sqs\SqsClient;
+use Illuminate\Queue\Connectors\ConnectorInterface;
+use Symfony\Component\HttpClient\HttpClient;
+
+class AsyncAwsSqsConnector implements ConnectorInterface
+{
+    /**
+     * Establish a queue connection.
+     *
+     * @return \Illuminate\Contracts\Queue\Queue
+     */
+    public function connect(array $config)
+    {
+        $clientConfig = [];
+        if ($config['key'] && $config['secret']) {
+            $clientConfig['accessKeyId'] = $config['key'] ?? null;
+            $clientConfig['accessKeySecret'] = $config['secret'] ?? null;
+            $clientConfig['sessionToken'] = $config['token'] ?? null;
+        }
+
+        return new AsyncAwsSqsQueue(
+            new SqsClient($clientConfig, null, HttpClient::create(['timeout' => 20])),
```

Comment:
'timeout' => 20 is maybe not enougth for a 20s polling (+ network latency), I suggest to use 30